src/lib/components/ShowMoreItems/ShowMoreItems.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'semantic-ui-react';
+
+export class ShowMoreItems extends Component {
+  state = {
+    expanded: false,
+  };
+
+  handleOnClick = e => {
+    e.preventDefault();
+    this.setState(state => {
+      return {
+        expanded: !state.expanded,
+      };
+    });
+  };
+
+  renderButton = () => {
+    const { expanded } = this.state;
+    const text = expanded ? 'Show less' : 'Show more';
```

Comment:
I noticed that we are using the `ShowMore` package:
`import ShowMore from 'react-show-more';`
is it something that can be reused here?

Reply:
Yes that's right, unfortunately this package is only meant to be used for text paragraph. I've tried out of curiosity with the list and while it 'works' (in the sense no error is thrown), the result is not what is intended.
There is however a package [`react-show-more-list`](https://www.npmjs.com/package/react-show-more-list) that seems to achieve that but given the download count it's probably not maintained and not guaranteed that it works.

So instead I reimplemented a component that provides this behavior for lists.